cg/meta/upload/balsamic/balsamic.py

```diff
@@ -36,5 +38,17 @@ def upload(self, ctx: click.Context, case_obj: models.Family, restart: bool) ->
         # Scout specific upload
         if DataDelivery.SCOUT in case_obj.data_delivery:
             ctx.invoke(scout, case_id=case_obj.internal_id, re_upload=restart)
+        else:
+            LOG.warning(
+                f"There is nothing to upload to Scout for case {case_obj.internal_id} and "
+                f"the specified data delivery ({case_obj.data_delivery})"
+            )
+
+        # Genotype specific upload
```

Comment:
```suggestion
```
I think this comment is unnecessary. It is evident on the next code block that it is specific for Genotype upload